deployment/updates/scripts/add_permissions/2016_02_11_add_qna_user_role_and_permissions.php

```diff
@@ -0,0 +1,32 @@
+<?php
+/**
+ * @package deployment
+ * @subpackage qna.roles_and_permissions
+ *
+ * Add qna user role and permission
+ */
+
+$script = realpath(dirname(__FILE__) . '/../../../../') . '/alpha/scripts/utils/permissions/addPermissionsAndItems.php';
+$config = realpath(dirname(__FILE__)) . '/../../../permissions/partner.0.ini';
+passthru("php $script $config");
+
+$script = realpath(dirname(__FILE__) . '/../../../') . '/base/scripts/insertDefaults.php';
+$config = realpath(dirname(__FILE__) . '/../../../') . '/base/scripts/init_data/03.UserRole.ini';
+passthru("php $script $config");
+
+
+$script = realpath(dirname(__FILE__) . '/../../../../') . '/alpha/scripts/utils/permissions/addPermissionsAndItems.php';
+$config = realpath(dirname(__FILE__)) . '/../../../permissions/object.KalturaAnnotationCuePoint.ini';
+passthru("php $script $config");
+
+$script = realpath(dirname(__FILE__) . '/../../../../') . '/alpha/scripts/utils/permissions/addPermissionsAndItems.php';
+$config = realpath(dirname(__FILE__)) . '/../../../permissions/object.KalturaWidget.ini';
+passthru("php $script $config");
+
+$script = realpath(dirname(__FILE__) . '/../../../../') . '/alpha/scripts/utils/permissions/addPermissionsAndItems.php';
+$config = realpath(dirname(__FILE__)) . '/../../../permissions/service.metadata.metadata.ini';
+passthru("php $script $config");
+
+$script = realpath(dirname(__FILE__) . '/../../../../') . '/alpha/scripts/utils/permissions/addPermissionsAndItems.php';
```

Comment:
I don't see this file was changed: service.metadata.metadataprofile
no need to run